flow/controllers/lane_change_controllers.py

```diff
@@ -24,3 +25,53 @@ class StaticLaneChanger(BaseLaneChangeController):
     def get_lane_change_action(self, env):
         """See parent class."""
         return 0
+
+
+class SafeAggressiveLaneChanger(BaseLaneChangeController):
+    def __init__(self, veh_id, target_velocity, threshold=0.75, desired_lc_time_headway=2.0):
+        """
+        A lane-changing model used to perpetually keep a vehicle in the same
+        lane.
```

Comment:
This docstring is confusing, wouldn't you just turn lane changing off?

Reply:
oh didn't update this. will fix